docs/OwnershipManifesto.md

```diff
@@ -1381,7 +1381,7 @@ comes from two main sources:
 Otherwise, the feature itself is pretty small.  The compiler
 implicitly emits moves instead of copies, just like we discussed
 above for the `move` intrinsic, and then diagnoses anything
-that that didn't work for.
+that didn't work with the `move` semantics.
```

Comment:
You can probably drop this part of the sentence entirely.  DI's involvement in enforcement is mentioned above already.